charts/gardener-extension-networking-calico/templates/clusterfilters-logging.yaml

```diff
@@ -3,6 +3,7 @@ kind: ClusterFilter
 metadata:
   labels:
     fluentbit.gardener/type: "seed"
+    fluentbit.gardener.cloud/type: "seed"
```

Comment:
`fluentbit.gardener/type: "seed"` was a mistake made by the one who introduced the fluent-bit operator.
Because we were in a hurry we decided to not change the annotation until the fluent-bit operator PR is merged in g/g. Now when is already on live we make preparation to fix this issue. Still the gardelet is working with `fluentbit.gardener/type` but in the future will switch to `fluentbit.gardener.cloud/type`.

Reply:
So you are essentially saying that currently `fluentbit.gardener/type` is used, but in future ` fluentbit.gardener.cloud/type` will be used.

Is that correct?

Reply:
yes